full-service/src/service/txo.rs

```diff
@@ -86,6 +114,57 @@ where
         Ok(Txo::get(&txo_id.to_string(), &conn)?)
     }
 
+    fn split_txo(
+        &self,
+        txo_id: &TxoID,
+        output_values: &[String],
+        subaddress_index: Option<i64>,
+        fee: Option<String>,
+        tombstone_block: Option<String>,
+    ) -> Result<(TransactionLog, AssociatedTxos), TxoServiceError> {
+        use crate::service::txo::TxoServiceError::{TxoNotSpendable, TxoNotSpendableByAnyAccount};
+
+        let conn = self.wallet_db.get_conn()?;
+        let txo_details = Txo::get(&txo_id.to_string(), &conn)?;
```

Comment:
I think this should all go into a db transaction

Reply:
👍